include/mbedtls/md.h

```diff
@@ -132,220 +138,255 @@ void mbedtls_md_free( mbedtls_md_context_t *ctx );
- * \param src       The context to be cloned
+ * \param dst       The destination context.
+ * \param src       The context to be cloned.
  *
  * \return          \c 0 on success,
- *                  \c MBEDTLS_ERR_MD_BAD_INPUT_DATA on parameter failure.
+ *                  #MBEDTLS_ERR_MD_BAD_INPUT_DATA on parameter failure.
  */
 int mbedtls_md_clone( mbedtls_md_context_t *dst,
                       const mbedtls_md_context_t *src );
 
 /**
- * \brief           Returns the size of the message digest output.
+ * \brief           This function returns the size of the message-digest output.
  *
- * \param md_info   message digest info
+ * \param md_info   The message-digest information.
  *
- * \return          size of the message digest output in bytes.
+ * \return          The size of the message-digest output in Bytes.
  */
 unsigned char mbedtls_md_get_size( const mbedtls_md_info_t *md_info );
 
 /**
- * \brief           Returns the type of the message digest output.
+ * \brief           This function returns the type of the message-digest output.
  *
- * \param md_info   message digest info
+ * \param md_info   The message-digest information.
  *
- * \return          type of the message digest output.
+ * \return          The type of the message-digest output.
  */
 mbedtls_md_type_t mbedtls_md_get_type( const mbedtls_md_info_t *md_info );
 
 /**
- * \brief           Returns the name of the message digest output.
+ * \brief           This function returns the name of the message-digest output.
  *
- * \param md_info   message digest info
+ * \param md_info   The message-digest information.
  *
- * \return          name of the message digest output.
+ * \return          The name of the message-digest output.
  */
 const char *mbedtls_md_get_name( const mbedtls_md_info_t *md_info );
 
 /**
- * \brief           Prepare the context to digest a new message.
- *                  Generally called after mbedtls_md_setup() or mbedtls_md_finish().
- *                  Followed by mbedtls_md_update().
+ * \brief           This function prepares the message-digest context for  
+ *                  a new message. 
  *
- * \param ctx       generic message digest context.
+ *                  it is generally called after mbedtls_md_setup() or 
+ *                  mbedtls_md_finish(). It is followed by mbedtls_md_update().
  *
- * \returns         0 on success, MBEDTLS_ERR_MD_BAD_INPUT_DATA if parameter
- *                  verification fails.
+ * \param ctx       The generic message-digest context.
+ *
+ * \returns         \c 0 on success, #MBEDTLS_ERR_MD_BAD_INPUT_DATA if 
+ *                  parameter verification fails.
  */
 int mbedtls_md_starts( mbedtls_md_context_t *ctx );
 
 /**
- * \brief           Generic message digest process buffer
- *                  Called between mbedtls_md_starts() and mbedtls_md_finish().
+ * \brief           This function performs a digest operation on 
+ *                  the input message.
```

Comment:
Suggestion: Follow the same wording as in `sha1.h`, i.e. `This function feeds an input buffer into an ongoing message-digest computation.`

Reply:
Fixed